server/src/Core/Validators/GameNameDuplicationsChecker.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using BGS.ApplicationCore.Entities;
+using BGS.ApplicationCore.Interfaces;
+using BGS.ApplicationCore.Specifications;
+using BGS.SharedKernel;
+
+namespace BGS.ApplicationCore.Validators;
+
+public class GameNameDuplicationsChecker(IRepository<Game> gameRepository) : IGameNameDuplicationsChecker
+{
+    public async Task<bool> Check(string gameName, CancellationToken cancellationToken)
+    {
+        return await gameRepository.AnyAsync(
+            new GameNameDuplicationsSpecification(gameName),
+            cancellationToken);
```

Comment:
don't forget about it - https://cezarypiatek.github.io/post/async-analyzers-p1/#1-redundant-asyncawait